src/h5web/visualizations/heatmap/interpolators.ts

```diff
@@ -38,25 +38,25 @@ import {
   interpolateTurbo,
 } from 'd3-scale-chromatic';
 
-export const INTERPOLATORS = {
-  // Diverging
-  BrBG: interpolateBrBG,
-  PRGn: interpolatePRGn,
-  PiYG: interpolatePiYG,
-  PuOr: interpolatePuOr,
-  RdBu: interpolateRdBu,
-  RdGy: interpolateRdGy,
-  RdYlBu: interpolateRdYlBu,
-  RdYlGn: interpolateRdYlGn,
+export const MOST_USED = {
+  Viridis: interpolateViridis,
+  Cividis: interpolateCividis,
+  Inferno: interpolateInferno,
+  Greys: interpolateGreys,
   Spectral: interpolateSpectral,
-  // Sequential (single hue)
+  Rainbow: interpolateRainbow,
+};
```

Comment:
The most used interpolators -- happy to tweak as needed, of course.

Reply:
I would not put `Viridis` **and** `Cividis` as both are quite similar. Also, for the diverging one, I would choose something less colorful than `Spectral`: `RdYlBu` or even better `RdBu`. The rest is fine to me but in any case, it will probably change once we get user feedback.

Reply:
Done 👍